test/test_utils.py

```diff
@@ -248,5 +258,64 @@ def test_draw_segmentation_masks_errors():
         utils.draw_segmentation_masks(image=img, masks=masks, colors=bad_colors)
 
 
+def test_draw_keypoints_vanilla():
```

Comment:
We just do like boxes, save an image and check after loading if expected image was same as saved image.

Reply:
Ok with me